### ISSUE TITLE
video start lagging when playing videos with audio track shorter than video track

### ISSUE THREAD

**Issue by agudpp** [wang-bin/QtAV#538]
Hi there :),
We detect an issue when playing videos that contains shorter length of audio track, they are not being played correctly with the lib (nor the player that you can download form the web page).

When using the lib we got the following messages (when the lags starts) from the **VideoThread**:

```
"delay 2.888786s @12.921905s"
"video too fast!!! sleep 0.19 s, nb fast: 39, v_a: 1.3440"
"delay 2.955655s @12.921905s"
"video too fast!!! sleep 0.20 s, nb fast: 40, v_a: 1.3776"
"delay 3.022256s @12.921905s"
"video too fast!!! sleep 0.20 s, nb fast: 41, v_a: 1.4108"
"delay 3.089223s @12.921905s"
"video too fast!!! sleep 0.21 s, nb fast: 42, v_a: 1.4445"
....
```

There is a sample video in here:

- http://we.tl/fHcGl2UJYA

Playing the video with other video players worked fine. 

For now the workaround we are doing is extending the sound with silence to the length of the video so the lagging doesn't appear.

Thanks again, let me know if I can help.

**Comment by wang-bin** [comment]
Hi, I forgot this issue. Can you upload your sample again?

**Comment by agudpp** [comment]
Hi want-bin, here it is attached.

[video_playback_bug.mp4.zip](https://github.com/wang-bin/QtAV/files/123475/video_playback_bug.mp4.zip)

**Comment by wang-bin** [comment]
@agudpp This bug should be fixed about 1 month ago 395b168 
Master branch plays fine

**Comment by agudpp** [comment]
@wang-bin  :+1: great :D! thanks!!
